src/scripts/crafttweaker/utils.zs

```diff
@@ -68,34 +90,19 @@ function formatBucket(bucket as IItemStack, liquidName as string) as IItemStack
 				Amount: 1000
 			}
 		};
+	} else if (bucket.matches(<immersiveengineering:jerrycan:0>)) {
+		data = {
+			Fluid: {
+				FluidName: liquidName
```

Comment:
No fluid amount? I think this would be the cause of the "broken" jerrycan in the recipe. See other buckets in the if statements.

Reply:
This might also be the cause of it accepting 1 mb

Reply:
Amount is deliberately omitted to allow crafting with various amounts in the Jerrycan. If an amount is specified it defeats the object of adding support for it.

Reply:
It should accept cans with more than 1,000 mb even with fluid amount as 1,000. You can see I've changed nothing in the code in SF4 https://github.com/Darkosto/SkyFactory4/blob/master/src/scripts/crafttweaker/utils.zs#L38-L74 but it now supports morebuckets (which some can have 32,000 mb)

Reply:
Also, this should be able to be simplified into the same case as the betweenland's buckets.

Reply:
That doesn't work in SF4 either...

Reply:
It will only accept a bucket from More Buckets with exactly 1000mB of whatever fluid the recipe requests.

![Screenshot 2019-06-06 at 20 01 50](https://user-images.githubusercontent.com/23245543/59059083-01e88b80-8896-11e9-95a3-90b711c1b568.png)
![Screenshot 2019-06-06 at 20 01 55](https://user-images.githubusercontent.com/23245543/59059086-044ae580-8896-11e9-9318-227b9ce6f22b.png)

![Screenshot 2019-06-06 at 20 01 31](https://user-images.githubusercontent.com/23245543/59059111-0e6ce400-8896-11e9-815f-5ddbd9539af2.png)
![Screenshot 2019-06-06 at 20 01 37](https://user-images.githubusercontent.com/23245543/59059115-1036a780-8896-11e9-9b03-297e97cc982f.png)